readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information
+Build: {{ build.id }}
+Project: {{ build.project }}
```

Comment:
Might also use the slug here instead of the name, since that can't change.

Reply:
Actually, this is the id, but the slug is a lot better to show, I'll need to change the API